widget/graphic.go

```diff
@@ -71,7 +151,10 @@ func (g *Graphic) PreferredSize() (int, int) {
 	g.init.Do()
 	if g.Image != nil {
 		return g.Image.Size()
```

Comment:
This method is deprecated. Please update to the new suggested method  .Bounds().Size()